tigerpath/majors_and_certificates/majors/FIT_2018.json

```
@@ -0,0 +1,178 @@
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-french-and-italian",
+    "https://fit.princeton.edu/undergraduate/concentrating/plan-study"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Thomas A. Trezise",
+      "email": "ttrezise@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Christy Wampole",
+      "email": "cwampole@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Prerequisites",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "Students who wish to concentrate in the department should possess a sound knowledge of either French or Italian by the end of their sophomore year. This can be achieved by completing the language sequence (FRE 107 or 108; ITA 107 or 108), or by demonstrating equivalent qualification. Students are also strongly encouraged to complete at least one advanced-language course (FRE 207, 207F or 208; ITA 207I, 208) prior to admission, or shortly thereafter, which can then be counted toward the concentration. FIT recommends that students complete at least one more 200-level course (such as FRE 211, 215, 217, 221, 222, 224, 225, 230, or ITA 208, 209, 220, 225) before enrolling in a 300-level course.",
+      "completed_by_semester": 4,
+      "req_list": [
+        {
+          "name": "Standard",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": "Students who wish to concentrate in the department should possess a sound knowledge of either French or Italian by the end of their sophomore year. This can be achieved by completing the language sequence (FRE 107 or 108; ITA 107 or 108), or by demonstrating equivalent qualification.",
+          "course_list": [
+            "FRE 107",
+            "FRE 108",
+            "ITA 107",
+            "ITA 108"
+          ]
+        },
+        {
+          "name": "Advanced",
+          "max_counted": 0,
+          "min_needed": null,
+          "explanation": "Students are also strongly encouraged to complete at least one advanced-language course (FRE 207, 207F or 208; ITA 207I, 208) prior to admission, or shortly thereafter, which can then be counted toward the concentration. FIT recommends that students complete at least one more 200-level course (such as FRE 211, 215, 217, 221, 222, 224, 225, 230, or ITA 208, 209, 220, 225) before enrolling in a 300-level course.",
+          "double_counting_allowed": true,
+          "course_list": [
+            "FRE 207",
+            "FRE 207F",
+            "FRE 208",
+            "FRE 211",
+            "FRE 215",
+            "FRE 217",
+            "FRE 221",
+            "FRE 222",
+            "FRE 224",

```

Missing FRE 225 and 230 (based on the department [website](https://fit.princeton.edu/undergraduate/concentrating/plan-study))
```suggestion
            "FRE 224",
            "FRE 225",
            "FRE 230",
```